framework/commercelayer/api/utils/normalizeLineItems.ts

```diff
@@ -1,3 +1,5 @@
+import data from '../../data.json'
```

Comment:
The schema should be built with items from the real services, otherwise it won't work when people deploy their apps.